withdraw-by-check/LETTERSPECS/BANNO.CHECK.WITHDRAW.V1.CFG

```diff
@@ -53,7 +53,7 @@ LT: ALL
 *
 * MINIMUM AND MAXIMUM WD AMOUNTS
 *  Line must begin with "MIN:" and/or "MAX:"
-*  Enter the minimum and/or maximum check amount for check withdraws
+*  Enter the minimum and/or maximum check amount for check withdrawls
 *  from Shares. Leaving either value blank denotes no min/max amount
 *  for that value.
 *  VALID VALUES: 0.01 - 999,999.99
```

Comment:
Should MIN:50.00 and MAX:80.00 be left blank, as the defaults are blank?

Reply:
Corrected